### PR TITLE
Update thirdPartyPreparerFullName and conditionally validate SoT

### PR DESCRIPTION
**Pull request by ndsprinkle**
## Summary
This updates form 21-0966 to support conditional logic in the statement of truth based on the identity of the person filling out the form. There is also an update to a property name in here, `thirdPartyPreparerFullName` to distinguish it more from the default `preparerFullName`

## Related issue(s)
department-of-veterans-affairs/va.gov-team-forms#647

### CODE REVIEW

**Review by tlei123 — APPROVED** [review]
LGTM, but if you wanna wait for my 0247 tests PR to get merged I've added a [getPagePaths() helper-method](https://github.com/department-of-veterans-affairs/vets-website/pull/25906/files#r1339462437) so we don't have to keep creating pagePaths file.

**Comment by ndsprinkle** [comment]
@tlei123 Thanks for pointing out that helper method. I'll loop back around in one of the later PRs and update that.

**Pull request merged**
